### PR TITLE
fix #318 Capitalized table and form fields

### LINKED ISSUE #318: [bug]: N should be capitalize in number field
Problem
In the payment invoice section when we look at the table then we can observe that
"N" should be capitalize in number field the ie "number"--->"Number".

How to reproduce
1.open the website
2.go to payment invoice section
3.observe the 1st field of the table

Here is the screenshot
image

### PR DESCRIPTION
**Pull request by Nam1001**
fix #318
Now Table and Form fields are properly capitalized

Here are some screenshots

image

image

### CODE REVIEW

**Comment by Nam1001** [comment]
hey @polymahh look at this pr

**Review by salahlalami — CHANGES_REQUESTED** [review]
@Nam1001 pls revert tmp.variables.env file

**Comment by Nam1001** [comment]
@polymahh 
@salahlalami  
hey i have added tmp.variable.env and also my PR will close issue #210 please review it and let me know

**Review by polymahh — APPROVED** [review]
@Nam1001 thank you, everything seems good now

File: frontend/src/forms/ٌRoleForm.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default function RoleForm({ isUpdateForm = false }) {
     <>
       <Form.Item
         name="codeName"
-        label="codeName"
+        label="CodeName"
```
Comment (Ando22) [inline frontend/src/forms/ٌRoleForm.jsx:9]: can you change this into `Code Name` instead of `CodeName`

Comment (Ando22) [reply frontend/src/forms/ٌRoleForm.jsx]: thank you!

**Review by Ando22 — APPROVED** [review]

**Comment by Nam1001** [comment]
@salahlalami please review this pr

**Review by adeojoemmanuel — APPROVED** [review]

**Review by salahlalami — APPROVED** [review]
@Nam1001 Thank you

**Pull request merged**